tests/test_rpm_verifier.py

```diff
@@ -80,6 +81,66 @@
             True,
             id="multiple images with unsigned rpms + fail if unsigned",
         ),
+        pytest.param(
+            [
+                ProcessedImage(
+                    image="i1", unsigned_rpms=[], error="Failed to run command"
+                ),
+                ProcessedImage(image="i2", unsigned_rpms=["their-rpm"]),
+            ],
+            True,
+            dedent(
+                """
+                Found unsigned RPMs:
+                image: i2 | unsigned RPMs: their-rpm
+                Errors encountered:
+                i1: Failed to run command 
```

Comment:
I think we should make the format similar between the two failure reasons. Something like:
```suggestion
                Found unsigned RPMs:
                i2 | their-rpm
                Encountered errors:
                i1: Failed to run command 
```

Reply:
Done, PTAL